docker/Dockerfile

```diff
@@ -2,3 +2,6 @@
 FROM akeneo/pim-php-base:4.0
 COPY --from=composer:1 /usr/bin/composer /usr/local/bin/composer
 RUN chmod +x /usr/local/bin/composer
+RUN mkdir -p /etc/ssl/certs/
+ADD https://curl.haxx.se/ca/cacert.pem /etc/ssl/certs/ca-bundle.pem
```

Comment:
Any reason to prefer sourcing an external file like this?
I think we just need `apt-get install -y ca-certificates`?

Reply:
The only reason is ignorance! TIL! I'll try it out.

Reply:
Fixed.